# HG changeset patch
# User Torsten <mttl@mailbox.org>
# Date 1502568951 -7200
#      Sat Aug 12 22:15:51 2017 +0200
# Node ID 53dfc2329ba9495fb1b11ddd30a57bdd8cbd9498
# Parent  b590c674e27df1da7b8cac3e7f291d7266e19e67
* parser.cc (info_to_html): update according to new Menu-string

diff --git a/libgui/src/qtinfo/parser.cc b/libgui/src/qtinfo/parser.cc
--- a/libgui/src/qtinfo/parser.cc
+++ b/libgui/src/qtinfo/parser.cc
@@ -372,17 +372,17 @@ replace_colons (QString& text)
 
 static void
 info_to_html (QString& text)
 {
   text.replace ("&", "&amp;");
   text.replace ("<", "&lt;");
   text.replace (">", "&gt;");
 
-  text.replace ("\n *Menu:",
+  text.replace ("\n* Menu:",
                 "\n<font style=\"color:DarkRed;font-weight:bold\">Menu:</font>");
   text.replace ("See also:",
                 R"(<font style="color:DarkRed;font-style:italic;font-weight:bold">See also:</font>)");
   replace_links (text);
   replace_colons (text);
 }
 
 QString
